### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by afshinfard**
Quick instruction on calculating assembly-like metrics for the physical map.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ cd experiment
 bin/physlr-make physical-map lr=linkedreads ref=reference protocol=stlfr
 ```
 
+If you provide a reference genome, Physlr automatically maps the physical map to the input reference and outputs a PAF file. You can run the following command which uses the PAF file to calculate assembly-like quality metrics for the physical map.
```
Comment (lcoombe) [inline README.md:55]: I'd rearrange these sentences to make it more clear that providing a reference allows physlr to calculate the quality metrics for physical map. 
Something like:

If you provide a reference genome (with `ref`), physlr automatically calculates assembly-like quality metrics for the physical map. Physlr automatically maps the physical map to the input reference, and outputs a PAF file that is parsed to calculate these metrics

File: README.md [hunk 2]
```diff
@@ -52,6 +52,12 @@ cd experiment
...
+```
+bin/physlr-make linkedreads.physlr.physical-map.reference.n10.qpos.chain.metrics.tsv lr=linkedreads ref=reference
```
Comment (lcoombe) [inline README.md:57]: The filename for this target is dependent on the `lr` and `ref` setting right? Might be easier to make an a 'phony' target called 'ref-metrics' or something like that in the Makefile, so a user wouldn't have to manually adjust the filename?

Comment (afshinfard) [reply README.md]: Done; with the current code Physlr generates this file automatically but one can still run this (with a phony target). Please see updates.

File: README.md [hunk 3]
```diff
@@ -44,14 +44,20 @@ cd physlr/src && make install PREFIX=/opt/physlr
 
 ## Generating Physlr Physical Map with stLFR reads
 
-To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
+To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome (optional).
```
Comment (jowong4) [inline README.md:47]: Is the "optional" needed?

Comment (afshinfard) [reply README.md:47]: @jowong4 I wanted to somehow remind the user that this is done _de novo_ and the reference is optional for those purposes mentioned. We can alternatively rephrase and add _de novo_ somewhere and remove this.

Comment (lcoombe) [reply README.md:47]: On that train of thought - I wonder if we are best to only have the first sentence here, and an example command _without_ a reference genome. Then, after that, we could say that you can visualize stats with a reference, and show an example of running in that way? 
Just thinking that we want to make it clear how to run physlr without a reference, which may be the more common use-case. Don't want someone thinking that it's reference-based.

**Comment by lcoombe** [comment]
@afshinfard - I just wasn't clear looking at this PR - Is generating the map quality stats currently default when running physlr? 
If so, how would a user run `physlr` without a reference genome?

**Comment by afshinfard** [comment]
@lcoombe So, the user can define a `ref` or not. It is optional. If the input the `ref`, then Physlr generates a physical-map and a metrics file and some visualizations. If they don't, then only a physical-map will be generated and they can later generate those other files using the `map-quality` phony by providing a reference.

**Comment by lcoombe** [comment]
> @lcoombe So, the user can define a `ref` or not. It is optional. If the input the `ref`, then Physlr generates a physical-map and a metrics file and some visualizations. If they don't, then only a physical-map will be generated and they can later generate those other files using the `map-quality` phony by providing a reference.

Great! I think that's how it should be. I think just adding in a relevant example showing launching physlr _without_ a reference in the README is still a good idea - because I wasn't sure from just looking at the README.

**Comment by afshinfard** [comment]
@lcoombe Good point, I agree, will change the example in the README to come with and without a `ref` for the `physical-map`

File: README.md (hunk 3 above)
Comment (afshinfard) [reply README.md]: Yeah, I was just editing and I was thinking the same way you just said... I'll switch to that one in a moment

Comment (afshinfard) [reply README.md]: @lcoombe done, you can check the changes now, thank you

**Comment by afshinfard** [comment]
sorry for the many small changes; it's ready to be reviewd.

File: README.md [hunk 4]
```diff
@@ -44,19 +44,40 @@ cd physlr/src && make install PREFIX=/opt/physlr
...
 ## Generating Physlr Physical Map with stLFR reads
 
-To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
-In this example, the linked reads and reference genome are called `linkedreads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `protocol=stlfr` to use the default value for stLFR reads.
+To construct a physical map _de novo_, you need linked reads (from 10X Genomics or MGI stLFR). 
 
+In this example, the linked reads dataset is called `linkedreads.fq.gz`. The linked reads are from stLFR so we specify `protocol=stlfr` to use the default value for stLFR reads.
+
+```
+cd experiment
```
Comment (lcoombe) [inline README.md:52]:
```suggestion
cd experiment # Change to working directory 
```
Just thinking to make it extra clear what you mean by this line?

**Review by lcoombe — APPROVED** [review]
Look good! I think it's nice and clear now :)
Just one more small suggestion from me.

File: README.md (hunk 4 above)
Comment (afshinfard) [reply README.md]: I agree, thank you for pointing out that things were unclear

**Pull request merged**